SQL/0000-00-00-schema.sql

```diff
@@ -754,6 +759,98 @@ CREATE TABLE `mri_protocol_checks` (
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
 
+-- ********************************
+-- BIDS tables
+-- ********************************
+
+CREATE TABLE `bids_category` (
+ `BIDSCategoryID`   int(3)      NOT NULL AUTO_INCREMENT,
+ `BIDSCategoryName` varchar(10) NOT NULL UNIQUE,
+ PRIMARY KEY (`BIDSCategoryID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+INSERT INTO `bids_category` (ImagingCategory) VALUES
+      ('anat'),
+      ('func'),
+      ('dwi'),
+      ('fmap');
+
+CREATE TABLE `bids_scan_type_subcategory` (
+  `BIDSScanTypeSubCategoryID` int(3)       NOT NULL AUTO_INCREMENT,
+  `BIDSScanTypeSubCategory`   varchar(100) NOT NULL UNIQUE,
+  PRIMARY KEY (`BIDSScanTypeSubCategoryID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+INSERT INTO `bids_scan_type_subcategory` (BIDSScanTypeSubCategory) VALUES
+  ('task-rest');
+
+CREATE TABLE `bids_scan_type` (
+  `BIDSScanTypeID` int(3)       NOT NULL AUTO_INCREMENT,
+  `BIDSScanType`   varchar(100) NOT NULL UNIQUE,
+  PRIMARY KEY (`BIDSScanTypeID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+INSERT INTO `bids_scan_type` (BIDSScanType) VALUES
+  ('bold'),
+  ('FLAIR'),
+  ('T1w'),
+  ('T2w'),
+  ('dwi');
+
+CREATE TABLE `bids_mri_scan_type_rel` (
+  `MRIScanTypeID`             int(10) UNSIGNED NOT NULL,
+  `BIDSCategoryID`            int(3)           DEFAULT NULL,
+  `BIDSScanTypeSubCategoryID` int(3)           DEFAULT NULL,
+  `BIDSScanTypeID`            int(3)           DEFAULT NULL,
+  `BIDSEchoNumber`            int(3)           DEFAULT NULL,
```

Comment:
all these IDs should be unsigned (don't forget the referred tables)  
except maybe BIDSEchoNumber. Can we have negative echo number?

Also, not sure exactly after reading past conversation but about BIDSEchoNumber, will the be a single value for each MRIScanTypeID or there might be multiple value for a MRIScanTypeID? It is important for the PK field list.